conftest.py

```diff
@@ -192,12 +193,25 @@ def setup_artifact_path(item: pytest.Item, artifact_name: str):
     return os.path.join(output_dir, artifact_name)
 
 
-def get_task_files_for_id(task_id: str) -> set:
+def get_task_files_for_id(task_id: str) -> dict:
     try:
-        return set(subprocess.check_output(['dcos', 'task', 'ls', task_id, '--all']).decode().split())
+        # Example output:
+        # drwxr-xr-x  6  nobody  nobody      4096  Jul 21 22:07                             jre1.8.0_144
+        # drwxr-xr-x  3  nobody  nobody      4096  Jun 28 12:50                          libmesos-bundle
+        # -rw-r--r--  1  nobody  nobody  32539549  Jan 04 16:31  libmesos-bundle-1.10-1.4-63e0814.tar.gz
+        ls_lines = subprocess.check_output(['dcos', 'task', 'ls', task_id, '--long', '--all']).decode().split('\n')
```

Comment:
Looking at the `ls --help` output, it may be useful to provide a timestamp format using:
```
   --time-style=STYLE     with -l, show times using style STYLE:
                               full-iso, long-iso, iso, locale, or +FORMAT;
                               FORMAT is interpreted like in 'date'; if FORMAT
                               is FORMAT1<newline>FORMAT2, then FORMAT1 applies
                               to non-recent files and FORMAT2 to recent files;
                               if STYLE is prefixed with 'posix-', STYLE
                               takes effect only outside the POSIX locale
```
Thus: `--time-style=+%m%d_%H%M` would get the timestamp in the desired format.

Reply:
(OK, just realized that this isn't calling ls ...)